### PR TITLE
[wip] improving a bit of this and that

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

**Comment by linuxbandit** [comment]
there's nothing to review now, I just had a bug in the cli and could not see which files are conflicting

File: oms.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ service_string=$(printenv ENABLED_SERVICES)
 ## Split services into array
 services=(${service_string//:/ })
 
-command="docker-compose -f base-docker-compose.yml"
+command="sudo docker-compose -f base-docker-compose.yml"
```
Comment (serge1peshcoff) [inline oms.sh:25]: Just for information, you can run Docker for non-sudo users: https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo
Not sure what be more secure though.

**Comment by linuxbandit** [comment]
this is closed as PR because it works better as local branch of upstream. Also I will improve the commit message which was just a fast wip

File: oms.sh (hunk 1 above)
Comment (linuxbandit) [reply oms.sh:25]: thanks! I will think about it in the optics of continuous deployment, because right now I am just thinking development

**Pull request closed without merging**